argo_rollouts/argo-rollouts-overview.adoc

```diff
@@ -0,0 +1,43 @@
+:_mod-docs-content-type: ASSEMBLY
+include::_attributes/common-attributes.adoc[]
+[id="argo-rollouts-overview"]
+= Argo Rollouts overview
+:context: argo-rollouts-overview
+
+toc::[]
+
+Progressive delivery is the process of releasing product updates in a controlled and gradual manner.
```

Comment:
This whole section is a massive wall of text. Consider breaking it up with bullets and subheadings, maybe a table if applicable. Benefits are also given in paragraphs here and in the Benefits selection below; and who benefits from what is all mashed together. Very hard to read an find information.

Since there are a few other nits, perhaps you could throw in some bullets?